canopen/node/local.py

```diff
@@ -127,3 +138,32 @@ def _find_object(self, index, subindex):
                 raise SdoAbortedError(0x06090011)
             obj = obj[subindex]
         return obj
+
+    def _nmt_state_changed(self, old_state, new_state):
+        if new_state == "OPERATIONAL":
+            for pdo in self.tpdo.map.values():
+                if pdo.enabled:
+                    try:
+                        pdo.start()
+                        logger.info("Successfully started %s", pdo.name)
+                    except ValueError:
+                        logger.warning("Failed to start %s due to missing period", pdo.name)
+                    except Exception:
+                        logger.exception("Unknown error starting %s", pdo.name)
+                else:
+                    logger.info("%s not enabled", pdo.name)
```

Comment:
Now this logging is probably too much.  Worst case, it will be emitted for all 512 PDOs.  That's too much even on DEBUG level.  I'd just drop this log call.